### DISCUSSION TITLE
Sync schema changes [Nov 09]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4410] [Discord Threads]
> 🙋‍♂️ **emre_**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### https://docs.nocodb.com/setup-and-usages/sync-schema  
#  
> 🙋‍♂️ **emre_**  
#### That doesn't work for me and shows as it's synced but i don't see any tables synced in boths sides. it looks like it's not attached at all  
#  
> 💁 **o1lab**  
#### ya try to recreate the proejct   
#### ensure the details are fine  
#  
> 🙋‍♂️ **emre_**  
#### already tried 3 times. is there any way to test it somehow on heroku side? or to see where the created nocodb tables are hosted?  
#  
> 🙋‍♂️ **emre_**  
#### i check the connection, it says it's fine. but when i create the project not attached at all 😃  
#  
> 🙋‍♂️ **emre_**  
#### Okay o1lab@discord i managed to attach them. but another rookie question. what are those pgstatements and can i hide them?  
![image0](https://cdn.discordapp.com/attachments/956279429968056351/956284219162779738/SCR-20220323-tfj.png)  
#  
> 🙋‍♂️ **emre_**  
#### Thanks for your fast response and help ❤️  
#  
> 💁 **o1lab**  
#### https://docs.nocodb.com/setup-and-usages/meta-management#ui-access-control  
#  
> 🙋‍♂️ **peehhhh**  
#### how did you managed to fix it? I'm having the same problem.  
#  
> 🙋‍♂️ **peehhhh**  
#### I'm hosting on Render and external database on Supabase. in Supabase connection logs I can see the requests coming from NocoDB. But no table shows up.  If I delete everything and start over and connect to same external db(supabase) I can still see the table I created from nocoui in the last run.  
#  
> 🙋‍♂️ **emre_**  
#### tbh i don't remember. after a while i completely abandoned it. not using nocodb atm. handle all my work inside supabase. <@380403684099751946>  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/956279429968056351)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
